### PR TITLE
TDI-37812 Snowflake component tests are not properly isolated.

### PR DESCRIPTION
**Pull request by francisu**
**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [X] Tests for the changes have been added (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

Tests fail when run concurrently because they used the same schema.

**What is the new behavior?**

Tests work when run concurrently (tested 3 concurrent tests)

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

A couple of other small corrections. Also removed the running of the plain tests (by making it an abstract class), only the Spring and OSGi tests are being used.

### CODE REVIEW

File: components-api/src/test/java/org/talend/components/api/properties/ComponentPropertiesTest.java [hunk 1]
```diff
@@ -75,6 +76,7 @@ public void init() {
     @Test
     public void testUpdateNestedProperties() throws IllegalAccessException {
         NestedNestedProperties nestedProperties = new NestedNestedProperties("bar");
+        System.out.println(SerializerDeserializer.toSerializedPersistent(nestedProperties));
         nestedProperties.value.setValue("XYZ");
```
Comment (francisu) [inline components-api/src/test/java/org/talend/components/api/properties/ComponentPropertiesTest.java]: Not really part of the change; but put in to show someone something. I think leaving it is harmless.

Comment (sgandon) [inline components-api/src/test/java/org/talend/components/api/properties/ComponentPropertiesTest.java]: why don't you use a slf4j Logger ?

**Comment by sgandon** [comment]
LGTM
